app/__init__.py

```diff
@@ -46,16 +49,36 @@ def shutdown_session(exception=None):
         db.session.remove()
 
 
-def create_app(config):
+from config import config_dict
+
+DEBUG = db_config("DEBUG", default=True)
+get_config_mode = "Debug" if DEBUG else "Production"
+app_config = config_dict[get_config_mode.capitalize()]
+
+celery_beat_schedule = {
+    "call-every-10": {
```

Comment:
perhaps we could give the key a more useful name that gives a relation to what the task does?

Reply:
I will just remove it since it's not necessary for now.

Reply:
okay